frontend/server/src/Controllers/User.php

```diff
@@ -167,6 +167,40 @@ public static function createUser(
             'verification_id' => \OmegaUp\SecurityTools::randomString(50),
             'is_private' => boolval($createUserParams->isPrivate),
         ];
+        if (
+            $createUserParams->birthDate >= strtotime(
+                '-13 year',
+                \OmegaUp\Time::get()
+            )
+        ) {
```

Comment:
Here we need to validate `parentEmail` param is not empty:
```suggestion
        if (
            $createUserParams->birthDate >= strtotime(
                '-13 year',
                \OmegaUp\Time::get()
            )
            && !empty($createUserParams->parentEmail)
        ) {
```

Reply:
Sure! I got the point.